### DISCUSSION TITLE
Proposed block will be handled in the future err="block in the future"

### DISCUSSION THREAD

**Discussion by sowjanya-jpmc** [ConsenSys/quorum#1548] [General]
Hi,

  I have 4 node quorum network. Peers were added one after another and added as validators using istanbul.propose(). After adding 4 node as validator, I see the below msg in node.log. What does it mean, node doesn't accept the blocks from a particular node?
Please let me know how to resolve this .

**INFO [11-04|05:46:55.999] Proposed block will be handled in the future address=0x6a65cef29d8a28b7E6223644ff81ceC3C2Ba483d from=0x87064D5396870956bE9C9f878530Ab4088afbb68 state="Accept request" err="block in the future" duration=497.582µs**
INFO [11-04|05:46:56.002] Committed                                address=0x6a65cef29d8a28b7E6223644ff81ceC3C2Ba483d hash=8cfb07…910188 number=7860
INFO [11-04|05:46:56.004] Imported new chain segment               blocks=1  txs=0 mgas=0.000 elapsed=1.134ms   mgasps=0.000 number=7860 hash=8cfb07…910188 dirty=0.00B
INFO [11-04|05:46:56.004] Commit new mining work                   number=7861 sealhash=90da15…aeaed5 uncles=0 txs=0 gas=0 fees=0 elapsed=251.684µs
INFO [11-04|05:47:01.002] Committed                                address=0x6a65cef29d8a28b7E6223644ff81ceC3C2Ba483d hash=939f97…0d9a37 number=7861
INFO [11-04|05:47:01.004] Imported new chain segment               blocks=1  txs=0 mgas=0.000 elapsed=916.004µs mgasps=0.000 number=7861 hash=939f97…0d9a37 dirty=0.00B
INFO [11-04|05:47:01.005] Commit new mining work                   number=7862 sealhash=eae0d2…96da56 uncles=0 txs=0 gas=0 fees=0 elapsed=187.729µs
INFO [11-04|05:47:06.003] Committed                                address=0x6a65cef29d8a28b7E6223644ff81ceC3C2Ba483d hash=9f4836…340720 number=7862
INFO [11-04|05:47:06.005] Imported new chain segment               blocks=1  txs=0 mgas=0.000 elapsed=874.4µs   mgasps=0.000 number=7862 hash=9f4836…340720 dirty=0.00B

**Comment by baptiste-b-pegasys** [comment]
The proposed block has the time when it is supposed be validated.
It's checked with the local clock of the validator.
So the clocks should be synced. Check if they are. (not sure we truncate at the higher second)

You can adjust the verification with `AllowedFutureBlockTime` in seconds, default is 0s.

**Reply by baptiste-b-pegasys** [reply]
https://consensys.net/docs/goquorum/en/latest/reference/cli-syntax/?h=allowedfutureblocktime#allowedfutureblocktime
